src/main/java/io/th0rgal/oraxen/items/OraxenMeta.java

```diff
@@ -51,6 +53,7 @@ public void setPackInfos(ConfigurationSection configurationSection) {
         // If not specified, check if a model or texture is set
         this.generate_model = configurationSection.getBoolean("generate_model", getModelName().isEmpty());
         this.parentModel = configurationSection.getString("parent_model", "item/generated");
+        this.modelDirectory = configurationSection.getString("model_directory", "minecraft/models");

```

Comment:
property should not be named `model_directory`as it is confusing
`generated_model_path` or somehting would be more self explanatory